### ISSUE TITLE
Custom dataset

### ISSUE THREAD

**Issue by Jorisfournel** [speedinghzl/pytorch-segmentation-toolbox#24]
Hi, 
Thanks for the contribution, could you show us what part of your code
is to be modified in order to train on a custom dataset. Also
in order to implement custom loss functions ?
Thanks in advance,
Joris

**Comment by speedinghzl** [comment]
In fact, the organization of this repo is clear and easy-to-read. The data loader is defined in file [dataset.py](https://github.com/speedinghzl/pytorch-segmentation-toolbox/blob/master/dataset/datasets.py) and loss functions are defined in [criterion.py](https://github.com/speedinghzl/pytorch-segmentation-toolbox/blob/master/utils/criterion.py).
